### PR TITLE
Adds updates for distribution channels

### PR DESCRIPTION
**Pull request by J-Byron**
## Problem
- Net needs to be updated to compensate for the incoming distribution channel changes

## Solution

- Removes config queries related to canonical_host
- Adds config for distribution channels site
- Bumps jekyll-contentful to <s>2.1.0</s> 3.0.0

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -87,18 +87,18 @@ collections:
     output: true
 
 contentful:
+  config:
+    sites: "www.crossroads.net"
```
Comment (tcmacdonald) [inline _config.yml]: Can you review the usage docs on this feature of jekyll-contentful? This does not appear to be an accurate use of this configuration value.

Comment (tcmacdonald) [reply _config.yml]: https://github.com/crdschurch/jekyll-contentful#content-distribution

**Comment by J-Byron** [comment]
> jekyll-contentful version did not change in this PR.

Looks like Jason updated jekyll-contenful in development https://github.com/crdschurch/crds-media/pull/831

**Comment by J-Byron** [comment]
Looks like Jason updated jekyll-contenful in development #831

**Pull request merged**